rails/app/apirest/config/routes.rb

```diff
@@ -0,0 +1,67 @@
+Myapp::Application.routes.draw do
+  root 'apicruds#index'
+
+  get 'show/:id', to: 'apicruds#show'
```

Comment:
É melhor usar a keyword `resources` que cria as rotas de acordo com a convenção: https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions

Caso não queira criar todas as rotas que o resource cria, é possível usar `except` com uma lista de symbols.